Client.Linq/Internal/Expressions/String/Contains.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class Contains : IExpressionPart
+    {
+        private readonly IEnumerable<IExpressionPart> expressionParts;
+        private readonly IEnumerable<IExpressionPart> expressionParts2;
+
+        internal Contains(IEnumerable<IExpressionPart> expressionParts, IEnumerable<IExpressionPart> expressionParts2)
+        {
+            this.expressionParts = expressionParts;
+            this.expressionParts2 = expressionParts2;
+        }
+
+        public void AppendFlux(StringBuilder builder)
+        {
+            builder.Append("strings.containsStr(v: ");
+            foreach (var expressionPart in expressionParts) expressionPart.AppendFlux(builder);
+            builder.Append(", substr: ");
+            foreach (var expressionPart in expressionParts2) expressionPart.AppendFlux(builder);
+            builder.Append(")");
+        }
```

Comment:
Please use same naming as is in Flux documentation:

```suggestion
        private readonly IEnumerable<IExpressionPart> _value;
        private readonly IEnumerable<IExpressionPart> _substr;

        internal Contains(IEnumerable<IExpressionPart> value, IEnumerable<IExpressionPart> substr)
        {
            _value = value;
            _substr = substr;
        }

        public void AppendFlux(StringBuilder builder)
        {
            builder.Append("strings.containsStr(v: ");
            foreach (var expressionPart in _value) expressionPart.AppendFlux(builder);
            builder.Append(", substr: ");
            foreach (var expressionPart in _substr) expressionPart.AppendFlux(builder);
            builder.Append(")");
        }
```